### ISSUE TITLE
[508-defect-3] CST: Improve link text for screenreader users

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#41817]
## Description

VA Claims Intake Center link text should include purpose e.g. mail them to to make the reason clear for screenreader users.

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/41447 (Under "Should" fix)

![VA claims intake center link text missing purpose](https://user-images.githubusercontent.com/14154792/169866040-b9f42887-7fdd-4f40-a776-e0a97aae5839.PNG)

## Tasks

- [x] Include "mail them to" text within the VA Claims Intake Center link
- [x] Add tests (as needed)

## Definition of done

- [x] All tasks complete
- [x] All tests passing
